src/vmm/src/cpu_config/x86_64/static_cpu_templates/t2s.rs

```diff
@@ -19,25 +20,55 @@ pub fn t2s() -> CustomCpuTemplate {
                 subleaf: 0x0,
                 flags: KvmCpuidFlags(0),
                 modifiers: vec![
+                    // EAX: Version Information
+                    // - Bits 03-00: Stepping ID.
+                    // - Bits 07-04: Model.
+                    // - Bits 11-08: Family.
+                    // - Bits 13-12: Processor Type.
+                    // - Bits 19-16: Extended Model ID.
+                    // - Bits 27-20: Extended Family ID.
                     CpuidRegisterModifier {
                         register: CpuidRegister::Eax,
                         bitmap: RegisterValueFilter {
-                            filter: 0b00001111111111110011111111111111,
-                            value: 0b00000000000000110000011011110010,
+                            filter: 0b0000_11111111_1111_00_11_1111_1111_1111,
+                            value: 0b0000_00000000_0011_00_00_0110_1111_0010,
                         },
                     },
+                    // ECX: Feature Information
+                    // - Bit 02: DTES64
+                    // - Bit 03: MONITOR
+                    // - Bit 04: DS-CPL
+                    // - Bit 05: VMX
+                    // - Bit 06: SMX
+                    // - Bit 07: EIST
+                    // - Bit 08: TM2
+                    // - Bit 10: CNXT-ID
+                    // - Bit 11: SDBG
+                    // - Bit 14: xTPR Update Control
+                    // - Bit 15: PDCM
+                    // - Bit 18: DCA
                     CpuidRegisterModifier {
                         register: CpuidRegister::Ecx,
                         bitmap: RegisterValueFilter {
-                            filter: 0b00000000000001001100110111111100,
-                            value: 0b00000000000000000000000000000000,
+                            filter: 0b0000_0000_0000_0100_1100_1101_1111_1100,
+                            value: 0b0000_0000_0000_0000_0000_0000_0000_0000,
                         },
                     },
+                    // EDX: Feature Information
+                    // - Bit 07: MCE
+                    // - Bit 12: MTRR
+                    // - Bit 18: PSN
+                    // - Bit 21: DS
+                    // - Bit 22: ACPI
+                    // - Bit 27: SS
+                    // - Bit 29: TM
+                    // - Bit 30: Reserved
```

Comment:
Just curious, any reason to specifically disable a reserved bit?

Reply:
Currently the bit is marked as reserved, but my guess is it was used for ia64 processor: https://en.wikipedia.org/wiki/CPUID

Reply:
Please see Itanium architecture SDM: https://www.intel.com/content/dam/www/public/us/en/documents/manuals/itanium-architecture-vol-4-manual.pdf